Makefile

```diff
@@ -37,10 +37,13 @@ temp_copy:
 	@GOPATH=$(ABS_TEMP_GOPATH) go get github.com/cossacklabs/acra/cmd/...
 
 
-build: temp_copy
+install: temp_copy
```

Comment:
here `make install` do not the same as in other our products. here it will copy binaries to `REPO_DIR/build` (https://github.com/cossacklabs/acra/blob/fa860a2f0f815165f954617985ef8ce1ae9129dd/Makefile#L4) when other our products place binaries to /usr/lib/*. I think we should install acra binaries to /usr/bin as default
and after changing path is important to drop this command - https://github.com/cossacklabs/acra/blob/fa860a2f0f815165f954617985ef8ce1ae9129dd/Makefile#L48 and explicitly list all binaries that should be deleted (acraproxy, acraserver, acra_genkeys, acra_rollback, acra_genpoisonrecord, acra_addzone) or it will delete `/usr/bin` folder when user will run `sudo make clean` :)